README.rst

```diff
@@ -1,16 +1,26 @@
-=========
-intbitset
-=========
+===========
+ intbitset
+===========
+
+.. image:: https://img.shields.io/travis/inveniosoftware/intbitset.svg
+        :target: https://travis-ci.org/inveniosoftware/intbitset
+
+.. image:: https://img.shields.io/coveralls/inveniosoftware/intbitset.svg
+        :target: https://coveralls.io/r/inveniosoftware/intbitset
```

Comment:
Yep, but not out of the box, so the coverage report will not be useful (and probably greyedout)